assembl/static2/package.json

```diff
@@ -9,7 +9,7 @@
   "scripts": {
     "test": "jest",
     "test:watch": "npm test -- --watch",
-    "build": "rm -rf node_modules/react-i18nify/node_modules;webpack --progress --colors",
+    "build": "webpack --progress --colors",
```

Comment:
This might need to be back in another environment. Only been tested on your debian, yeah?

Reply:
We now use a version of react-i18nify that doesn't use a hard coded react version. This `rm -rf` was needed to not have duplicate packages in the bundle.